### PR TITLE
Pre Install guide rework

### PR DESCRIPTION
**Pull request by gianmaticus**
Reworking mainly preinstall.md and adjusting other typos/formatting issues in docs.

### CODE REVIEW

File: docs/guides/preinstall.md
Comment (sharpenedblade) [inline docs/guides/preinstall.md:16]:
```suggestion
If there is an ISO with T2 support for your distro, you can download it here:
```

This change makes it much longer without improving clarity. The original sentence is already simple enough.

Comment (gianmaticus) [reply docs/guides/preinstall.md:16]:
```suggestion
Listed below are the currently available ISO files to download:
```
I think it's more well written than "If there is.."

Comment (sharpenedblade) [reply docs/guides/preinstall.md:16]: `Listed below are the currently available installer ISOs for download:` would be better

**Comment by sharpenedblade** [comment]
> I was under the impression of getting a PR similar to #446
> 
> Do you wish to add more changes?

I think they are doing it in multiple PRs

**Review by sharpenedblade — CHANGES_REQUESTED** [review]

File: docs/guides/preinstall.md
Comment (gianmaticus) [reply docs/guides/preinstall.md:29]: It seems a fair solution, considered the same link

**Review by sharpenedblade — CHANGES_REQUESTED** [review]

**Comment by AdityaGarg8** [comment]
Approving it for the sake of running lint

**Comment by AdityaGarg8** [comment]
Approving for lint again

**Comment by gianmaticus** [comment]
Dash typos in docs should be over. Fixed also the remaining lint issues.

**Comment by gianmaticus** [comment]
Lint issues should be over now.

File: docs/guides/preinstall.md
Comment (AdityaGarg8) [inline docs/guides/preinstall.md:74]: This change seems to unnecessary

File: docs/guides/preinstall.md
Comment (AdityaGarg8) [inline docs/guides/preinstall.md:52]: Except for the steps written down to change secure boot state, the other minor edits are uncessary. Also, you’ve removed the warning notes, why?

File: docs/guides/preinstall.md
Comment (AdityaGarg8) [inline docs/guides/preinstall.md:67]: Why removed try? Second one the generally the one, but in some cases the first one also works.

File: docs/guides/preinstall.md
Comment (AdityaGarg8) [inline docs/guides/preinstall.md:106]: What about the instruction stating to refer to the distros installation page?

File: docs/guides/preinstall.md
Comment (AdityaGarg8) [inline docs/guides/preinstall.md:120]: Again, a warning sign removed. I believe a warning box shall attract the eyes of a person more than a separate section written here.

**Review by AdityaGarg8 — CHANGES_REQUESTED** [review]
Most changes seem to make things complicated, anyways I’ve requested some changes for now, will have to do another review after they are made.

File: docs/guides/preinstall.md
Comment (gianmaticus) [reply docs/guides/preinstall.md:74]: Reverted the first part with some change to the "files" word repeated.
About the second one "Follow the instructions in this section to be followed" - _follow to be followed_ seems to me harder to read. 
I think the second part now is better clarified as in the commit [91c69da](https://github.com/gianmaticus/wiki/commit/91c69da74093a33156fd9bc6e03eec0984c86815)
 ·

File: docs/guides/preinstall.md
Comment (gianmaticus) [reply docs/guides/preinstall.md:52]: The warning note for "wait to finish creating your install usb" is already mentioned on line 67. I think it's redundant here after two steps.
The warning for "and you have copied Wi-Fi firmware" I also think it's redundant. Since it is already specified to come back on this guide again after finishing (line 73). 
That's why the !note has been removed in the commit.

File: docs/guides/preinstall.md
Comment (gianmaticus) [reply docs/guides/preinstall.md:67]: I think you are right, added try! [345547e](https://github.com/gianmaticus/wiki/commit/345547e92f485fc37c97359106ce0aa72ef19b95)

File: docs/guides/preinstall.md
Comment (gianmaticus) [reply docs/guides/preinstall.md]: I reverted the missing piece of the distro installation page. [89f231d](https://github.com/gianmaticus/wiki/commit/89f231dafc5bd679bcb3e0ac4be78c1bcd82d1e5)

File: docs/guides/preinstall.md
Comment (gianmaticus) [reply docs/guides/preinstall.md]: Reverted the warning sign. Just not sure if the formatting is right. I really want to make the mount points in bulleted list.

The recommendation to choose the distro in the overview has been removed, since there is already specified everything in the ISO choosing section. [61c3eb8](https://github.com/gianmaticus/wiki/commit/61c3eb8df1ad0637f6de48d4c2889cdb9dee2fe3)

File: docs/guides/preinstall.md
Comment (gianmaticus) [reply docs/guides/preinstall.md:71]: Reverted [df57217](https://github.com/gianmaticus/wiki/commit/df57217ff497820f13aaa8539c885d9a8c04dc75)

File: docs/guides/preinstall.md
Comment (gianmaticus) [reply docs/guides/preinstall.md:43]: Corrected ["may require" as suggested](https://github.com/t2linux/wiki/pull/447/commits/94234ab942f9b49a1b36a6eb1d14c97c30baa2df)

**Comment by gianmaticus** [comment]
> Most changes seem to make things complicated, anyways I’ve requested some changes for now, will have to do another review after they are made.

This seems also to me when comparing the git diff, but when downloading the commited .md and opening it in a text editor in non-code version, the reading of the guide is easier, more streamlined and some sentences are better expressed.

Git diff may be good for coding, but for text it may make you lose the "catch" behind the commit.
I noticed it as a first-time github user.

File: docs/guides/preinstall.md
Comment (AdityaGarg8) [inline docs/guides/preinstall.md:48]: Consider the fact that the wiki is not only about debian based distros, so the previous wording looks good to me.

File: docs/guides/preinstall.md
Comment (AdityaGarg8) [inline docs/guides/preinstall.md:57]: Or you can use *dd on* the macOS terminal:

File: docs/guides/preinstall.md
Comment (AdityaGarg8) [inline docs/guides/preinstall.md:73]: Can we simplify this line a bit? Follow the first part to be done in macOS of [this guide], maybe something like this.

File: docs/guides/preinstall.md
Comment (AdityaGarg8) [inline docs/guides/preinstall.md:84]: I don’t think “rapidly” word is required

**Review by AdityaGarg8 — CHANGES_REQUESTED** [review]

File: docs/guides/preinstall.md
Comment (gianmaticus) [reply docs/guides/preinstall.md:48]: That's why there is the link to the Basic setup guide (postinstall.md)
I think the linking of the two guides can replace those lines:
```
Please, refer to this guide: [Basic setup](https://wiki.t2linux.org/guides/postinstall/).
You can also find detailed info for unsupported debian-based distros [here](https://wiki.t2linux.org/distributions/debian/installation/).
```
Basic setup explains the procedure for non-T2 iso
More specifically there is a guide for debian-based non supported distro, where it is also mentioned about the efi=noruntime parameter.

What do you think?

File: docs/guides/preinstall.md
Comment (gianmaticus) [reply docs/guides/preinstall.md]: It sounds a lot better now: [12b7aa0](https://github.com/gianmaticus/wiki/commit/12b7aa0d90363d5052a341a5bb10283531e9d9e2)

File: docs/guides/preinstall.md
Comment (gianmaticus) [reply docs/guides/preinstall.md]: Removed [c7f57f2](https://github.com/t2linux/wiki/pull/447/commits/c7f57f2678d0ccc4500ffd1a4c80294b40962655)

File: docs/guides/preinstall.md
Comment (AdityaGarg8) [reply docs/guides/preinstall.md:48]: What I meant was that remove the word *debian-based*. Whatever you suggested for not supported distris is applicable for non-debian based unsupported distros as well.

**Review by AdityaGarg8 — CHANGES_REQUESTED** [review]

File: docs/guides/preinstall.md
Comment (AdityaGarg8) [reply docs/guides/preinstall.md]: I guess there is some grammatical error here.

File: docs/guides/preinstall.md
Comment (gianmaticus) [reply docs/guides/preinstall.md:48]: I just took it from the introduction: _This guide shall help you in installing a Debian or Ubuntu based Linux distro, which is not available as a modified ISO with T2 support._

What would like to modify?

File: docs/guides/preinstall.md
Comment (gianmaticus) [reply docs/guides/preinstall.md]: I can't find an error in my commit: [8596ecc](https://github.com/gianmaticus/wiki/commit/8596ecc87ae2969f82bf4aa3091a83f461cd12ef)
Where?

File: docs/guides/preinstall.md
Comment (gianmaticus) [reply docs/guides/preinstall.md:48]: > What I meant was that remove the word _debian-based_. Whatever you suggested for not supported distris is applicable for non-debian based unsupported distros as well.

Should be good now: [0cf920b](https://github.com/gianmaticus/wiki/commit/0cf920b809b0dacc44977be2f11e2ec09755abae)

Comment (AdityaGarg8) [reply docs/guides/preinstall.md:48]: Are we modifying the debian guide or preinstall guide. You've yourself added the link of the debian guide here, I believe the previous original one was better.

File: docs/guides/preinstall.md
Comment (AdityaGarg8) [inline docs/guides/preinstall.md:10]: Choose the volume you want to partition for Linux

This seems better, as the user needn't necessarily partition Macintosh HD. Also, if the user has renamed their macOS partition, then instead of Macintosh HD, the new name shall be there.

File: docs/guides/preinstall.md
Comment (AdityaGarg8) [reply docs/guides/preinstall.md:48]: Oh, I overlooked the heading, this seems good now.

File: docs/guides/preinstall.md
Comment (AdityaGarg8) [inline docs/guides/preinstall.md:114]: This change can be reverted

**Review by AdityaGarg8 — CHANGES_REQUESTED** [review]

File: docs/guides/preinstall.md
Comment (gianmaticus) [reply docs/guides/preinstall.md]: > Are we modifying the debian guide or preinstall guide. You've yourself added the link of the debian guide here, I believe the previous original one was better.
Reverted [e6e0838](https://github.com/gianmaticus/wiki/commit/e6e0838e5e772577acfea012b6bb449eec8301fd)

File: docs/guides/preinstall.md
Comment (gianmaticus) [reply docs/guides/preinstall.md]: Updated [4297a9b](https://github.com/gianmaticus/wiki/commit/4297a9bc7c5a0a832e88a59178fa8931fdbd07f3)

File: docs/guides/preinstall.md
Comment (AdityaGarg8) [reply docs/guides/preinstall.md:110]: You reverted the wrong commit. The change for debian was fine. You need to revert the change I've pointed out in this review.

**Review by AdityaGarg8 — CHANGES_REQUESTED** [review]

File: docs/guides/preinstall.md
Comment (gianmaticus) [reply docs/guides/preinstall.md]: reverted [1418a99](https://github.com/gianmaticus/wiki/commit/1418a99717e6f2a726d34932a8c4a6e8eaafbbe9)

**Comment by AdityaGarg8** [comment]
You'll have to revert https://github.com/t2linux/wiki/pull/447/commits/e6e0838e5e772577acfea012b6bb449eec8301fd as well to fix the wrong revert you did.

**Comment by AdityaGarg8** [comment]
The PR probably can be merged after the wrong revert is fixed.

**Comment by gianmaticus** [comment]
> You'll have to revert [e6e0838](https://github.com/t2linux/wiki/commit/e6e0838e5e772577acfea012b6bb449eec8301fd) as well to fix the wrong revert you did.

Check this: [a939310](https://github.com/gianmaticus/wiki/commit/a939310b4b17c6e14b62b0269a52e98a112432c9)

**Comment by AdityaGarg8** [comment]
Thanks for you contribution :)

**Review by AdityaGarg8 — APPROVED** [review]

**Pull request merged**
